package.json

```diff
@@ -40,12 +40,12 @@
     "suitcss-utils-display": "^0.4.0"
   },
   "scripts": {
-    "init": "npm run module && npm run virtualenv && npm run pip && npm install && npm run config",
+    "setup": "npm run module && npm run virtualenv && npm install && npm run config",
```

Comment:
+1 to updating right away the docs so it is in sync with whatever modification of the project.